apps/settings/js/panels/homescreen/homescreen.js

```diff
@@ -0,0 +1,35 @@
+define(function(require) {
+  'use strict';
+
+  var VerticalPreferences = require('shared/homescreens/vertical_preferences');
+  var Observable = require('modules/mvvm/observable');
+
+  var Homescreen = Observable({
```

Comment:
Please provide a function for setting the value of cols as the action is asynchronous.